query/app/src/main/resources/features.xml

```diff
@@ -80,6 +95,7 @@
              description="DDF :: Admin Console :: Query :: GraphQL">
         <feature>admin-query-core</feature>
         <feature>graphql-dependencies</feature>
+        <bundle>mvn:commons-collections/commons-collections/3.2.2</bundle>
```

Comment:
Same comment, there is no `thirdparty` feature for the app running in production.